docs/_deploying/heroku.md

```diff
@@ -0,0 +1,279 @@
+---
+title: Deploying to Heroku
+order: 0
+slug: heroku
+lang: en
+layout: tutorial
+permalink: /deploying/heroku
+redirect_from:
+  - /deploying
+---
+# Deploying to Heroku
+
+<div class="section-content">
+This guide will walk you through preparing and deploying a Slack app using [Bolt for JavaScript](/bolt-js) and the [Heroku platform](https://heroku.com/). Along the way, we’ll download a Bolt Slack app, prepare it for Heroku, and deploy it.
+</div>
+
+When you’re finished, you’ll have this ⚡️[Getting Started with Heroku app](https://github.com/mwbrooks/bolt-js-getting-started-with-heroku-app) to run, modify, and make your own.
+
+---
+
+### Get a Bolt Slack app
+
+We'll continue from [Getting Started with Bolt for JavaScript][getting-started-guide] to learn how to prepare and deploy a Slack app to Heroku.
+
+Download the Getting Started app and change into its directory:
+
+```shell
+git clone https://github.com/mwbrooks/bolt-js-getting-started-app.git
+cd bolt-js-getting-started-app/
+```
+
+> 💡 You also can follow along with an existing Bolt app and we'll mention any extra steps that are needed.
```

Comment:
I think you can remove this

Reply:
Agreed. Now that we have separated the two commands, it's more obvious that it's okay to use an existing app.

Fixed in 884c97c